www.chuyu.me/es.xml

```diff
@@ -1,6 +1,6 @@
 <Strings>
   <String Src="%lang%" Dst="es"/>
-  <String Src="%description%" Dst="Dism++ puede considerarse como una versión de Dism con interfaz gráfica, Limpieza de actualización, personalización de su imagen de sistema operativo, integración de actualizaciones y controladores."/>
+  <String Src="%description%" Dst="Dism++ puede considerarse como una versión de Dism con interfaz gráfica, Limpieza de actualización, Personalización de su imagen de sistema operativo, Integración de actualizaciones y controladores."/>
```

Comment:
This is correct, not you translation

Dism++ puede considerarse como una versión de Dism con interfaz gráfica, limpiando actualizaciones, personalizando la imagen del sistema operativo, integrando actualizaciones y controladores.

What is the difference?, the meaning is the same, after a comma cannot be capital letters